Languages/en_US/General.php

```diff
@@ -1136,12 +1136,12 @@
 }.';
 $txt['debug_query_in_line'] = 'in {file} on line {line}, ';
 $txt['debug_query_which_took'] = '{0, plural,
-	one {which took # second}
-	other {which took # seconds}
+	one {which took {0, number, ::precision-unlimited} second}
+	other {which took {0, number, ::precision-unlimited} seconds}
```

Comment:
```suggestion
	one {which took {0, number, :: .000 second}
	other {which took {0, number, :: .000} seconds}
```

This will give exactly three decimal places in the output every time.

Reply:
but 2.1 showed us all eight

![Screenshot 2024-03-27 at 18-37-26 Screenshot](https://github.com/SimpleMachines/SMF/assets/431799/eacf3d70-f6b6-42a7-8f1f-295f9c50d3a2)

Reply:
Then never mind. 🙂